common/config/src/lib.rs

```diff
@@ -196,6 +199,21 @@ impl Config {
 
         Config::try_from(raw_config)
     }
+
+    pub fn process_logdna_env_vars() {
+        std::env::vars_os()
+            .filter(|(n, _)| {
+                n.clone()
+                    .into_string()
+                    .unwrap_or_default()
+                    .starts_with(LOGDNA_PREFIX)
+            })
+            .for_each(|(name, value)| {
+                let new_name = MEZMO_PREFIX.to_string()
+                    + &name.into_string().unwrap_or_default()[LOGDNA_PREFIX.len()..];
+                std::env::set_var(new_name, value);
```

Comment:
I tried to work out a way of doing this without mutating the process environment, but I can't see a nice way to extend the StructOpt parsing to have multiple env vars. We could macro it, but it would pretty horrible. So this is probably the lesser evil